test/util/helpers.ts

```diff
@@ -121,11 +121,12 @@ export function buildMockParamsWithBlockParamAt(
  */
 export function buildMockParamsWithoutBlockParamAt(
   blockParamIndex: number,
+  value?: string,
```

Comment:
Do you want to revert these changes? It doesn't appear they are being used any longer.

Reply:
PR is updated